@app/db/scripts/dump-db.js

```diff
@@ -0,0 +1,30 @@
+const { spawn } = require("child_process");
+
+if (process.env.IN_TESTS === "1") {
+  process.exit(0);
+}
+
+const connectionString = process.env.GM_DBURL;
+if (!connectionString) {
+  console.error(
+    "This script should only be called from a graphile-migrate action."
+  );
+  process.exit(1);
+}
+
+spawn(
+  process.env.PG_DUMP || "pg_dump",
+  [
+    "--no-sync",
+    "--schema-only",
+    "--no-owner",
+    "--exclude-schema=graphile_migrate",
+    "--exclude-schema=graphile_worker",
+    "--file=../../data/schema.sql",
+    `"${connectionString}"`,
```

Comment:
I don't think the `"` quote marks are needed here; since `spawn` escapes all the arguments automatically. In fact, I think this will break the script on bash (not sure about Windows) because the quote marks will be escaped and included as part of the string. Should be able to just pass `connectionString` directly:

```suggestion
    connectionString,
```